Examples/Python/src/Geometry.cpp

```diff
@@ -103,19 +103,20 @@ void addGeometry(Context& ctx) {
     py::class_<Acts::GeometryIdentifier>(m, "GeometryIdentifier")
         .def(py::init<>())
         .def(py::init<Acts::GeometryIdentifier::Value>())
-        .def("setVolume", &Acts::GeometryIdentifier::setVolume)
-        .def("setLayer", &Acts::GeometryIdentifier::setLayer)
-        .def("setBoundary", &Acts::GeometryIdentifier::setBoundary)
-        .def("setApproach", &Acts::GeometryIdentifier::setApproach)
-        .def("setSensitive", &Acts::GeometryIdentifier::setSensitive)
-        .def("setExtra", &Acts::GeometryIdentifier::setExtra)
-        .def("volume", &Acts::GeometryIdentifier::volume)
-        .def("layer", &Acts::GeometryIdentifier::layer)
-        .def("boundary", &Acts::GeometryIdentifier::boundary)
-        .def("approach", &Acts::GeometryIdentifier::approach)
-        .def("sensitive", &Acts::GeometryIdentifier::sensitive)
-        .def("extra", &Acts::GeometryIdentifier::extra)
-        .def("value", &Acts::GeometryIdentifier::value)
+
+        .def_property("volume", &Acts::GeometryIdentifier::volume,
+                      &Acts::GeometryIdentifier::setVolume)
+        .def_property("layer", &Acts::GeometryIdentifier::layer,
+                      &Acts::GeometryIdentifier::setLayer)
+        .def_property("boundary", &Acts::GeometryIdentifier::boundary,
+                      &Acts::GeometryIdentifier::setBoundary)
+        .def_property("approach", &Acts::GeometryIdentifier::approach,
+                      &Acts::GeometryIdentifier::setApproach)
+        .def_property("sensitive", &Acts::GeometryIdentifier::sensitive,
+                      &Acts::GeometryIdentifier::setSensitive)
+        .def_property("extra", &Acts::GeometryIdentifier::extra,
+                      &Acts::GeometryIdentifier::setExtra)
+        .def_property_readonly("value", &Acts::GeometryIdentifier::value)
         .def("__str__", [](const Acts::GeometryIdentifier& self) {
```

Comment:
Do you think you could add a binding to `__repr__` as well in this PR? this would allow to easily `print` geometry ids. I guess it should be the same as `__str__`...
I have this in several branches, but always forget/am to lazy to make a PR...

Reply:
Doesn't `print` call `__str__` and not `__repr__`?